libbeat/docs/monitoring/shared-monitor-config.asciidoc

```diff
@@ -6,135 +6,117 @@
 //// Use the appropriate variables defined in the index.asciidoc file to
 //// resolve Beat names: beatname_uc and beatname_lc.
 //// Use the following include to pull this content into a doc file:
-//// include::../../libbeat/docs/monitoring/configuring.asciidoc[]
+//// include::../../libbeat/docs/monitoring/shared-monitor-config.asciidoc[]
 //// Make sure this content appears below a level 2 heading.
 //////////////////////////////////////////////////////////////////////////
 
+[role="xpack"]
 [[configuration-monitor]]
-== Monitoring configuration options
-++++
-<titleabbrev>Configuration options</titleabbrev>
-++++
+=== Settings for internal collection
 
-You can specify the following options in the `xpack.monitoring` section of the
+Use the following settings to configure internal collection.
+
+You specify these settings in the `monitoring` section of the
 +{beatname_lc}.yml+ config file:
 
-[float]
-=== `enabled`
+==== `enabled`
```

Comment:
In `master` this is worded slightly differently:

```
You specify these settings in the X-Pack monitoring section of the
+{beatname_lc}.yml+ config file:

==== `monitoring.enabled`
```

Any reason not to follow the same pattern in this PR?

Reply:
Not really, except for the reasons that I mentioned before (backporting all changes from 7.3 to present is more work and might introduce extra stuff).